code/modules/response_team/ert_outfits.dm

```diff
@@ -225,6 +225,7 @@
 	name = "RT Engineer (Amber)"
 	shoes = /obj/item/clothing/shoes/magboots
 	suit = /obj/item/clothing/suit/space/hardsuit/ert/engineer
+	belt = /obj/item/storage/belt/utility/chief/full
```

Comment:
Please change /datum/outfit/job/centcom/response_team/engineer's belt instead, and remove the belt var from amber and red